lib/find-visualstudio.js

```diff
@@ -113,6 +114,48 @@ class VisualStudioFinder {
     throw new Error('Could not find any Visual Studio installation to use')
   }
 
+  async findVisualStudio2017OrNewerUsingSetupModule (vcInstallDir) {
```

Comment:
Not required in this case, made it for any potential future parametrization or re-use. 
But can remove to avoid potential confusion and for consistency with the current approach.